### PR TITLE
[DOCS] Grouped navigation menu - JBP

### PR DESCRIPTION
**Pull request by joepeeples**
Addresses my portion of updates for the Grouped Navigation meta (#2237).

## Previews
Get started with Elastic Security
- [Configure and install the Endpoint and Cloud Security integration](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/install-endpoint.html)

Detections and alerts
- [Detections and Alerts](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/detection-engine-overview.html) - section intro
- [Create a detection rule](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html)
- [Manage detection rules](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-management.html)
- [Monitor and troubleshoot rule executions](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-monitor.html)
- [Rule exceptions and value lists](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/detections-ui-exceptions.html)
- [About building block rules](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/building-block-rule.html)
- [Manage detection alerts](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-manage.html)
- [Visual event analyzer](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/visual-event-analyzer.html)
- [Session View](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/session-view.html)
- [Tune detection rules](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/tuning-detection-signals.html)

Manage (Endpoint and Cloud Security)
- [Host isolation](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/host-isolation-ov.html)

API docs
- [Create rule](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/rules-api-create.html)
- [Update rule](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/rules-api-update.html)
- [Import rules](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/rules-api-import.html)
- [Export rules](https://security-docs_2244.docs-preview.app.elstc.co/guide/en/security/master/rules-api-import.html)

### CODE REVIEW

**Review by nastasha-solomon — APPROVED** [review]
I left one comment that'll apply to all other areas that have the same workflow. Other than that, it looks great. Thanks for updating all of these!

File: docs/detections/api/rules/rules-api-export.asciidoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ You cannot export prebuilt rules, but they are available at https://github.com/e
 =================
 Although detection rule actions are included in the exported file, the connectors used by the actions are not included. Use the {kibana-ref}/managing-saved-objects.html#managing-saved-objects-export-objects[Saved Objects] UI in Kibana (*Stack Management* -> *Kibana* -> *Saved Objects*) or the Saved Objects APIs (experimental) to {kibana-ref}/saved-objects-api-export.html[export] and {kibana-ref}/saved-objects-api-import.html[import] any necessary connectors _before_ you export and import the detection rules.
 
-Similarly, any value lists used for rule exceptions are not included in rule exports or imports. Use the <<edit-value-lists, Upload value lists>> UI (*Detect* -> *Rules* -> *Upload value lists*) to export and import value lists separately.
+Similarly, any value lists used for rule exceptions are not included in rule exports or imports. Use the <<edit-value-lists, Upload value lists>> UI (*Manage* -> *Rules* -> *Upload value lists*) to export and import value lists separately.
```
[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/detections/api/rules/rules-api-export.asciidoc:15]: Eagle-eyed catch @nastasha-solomon! Looks like they renamed it in 8.4. Created issue #2262 to fix it outside of this PR.

[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/detections/api/rules/rules-api-export.asciidoc:15]: 🦅 🦅 🦅

**Review by benironside — APPROVED** [review]
LGTM!

**Review by jmikell821 — APPROVED** [review]
So many nav changes! 😅  LGTM @joepeeples, thanks!

**Pull request merged**